tools/db_crashtest.py

```diff
@@ -793,14 +798,19 @@ def finalize_and_sanitize(src_params):
     if dest_params.get("create_timestamped_snapshot_one_in", 0) > 0:
         dest_params["txn_write_policy"] = 0
         dest_params["unordered_write"] = 0
-    # For TransactionDB, correctness testing with unsync data loss is currently
-    # compatible with only write committed policy
+    # For TransactionDB, correctness testing with unsync data loss and
+    # GetUpdateSince() are currently compatible with only write committed policy
     if dest_params.get("use_txn") == 1 and dest_params.get("txn_write_policy", 0) != 0:
         dest_params["sync_fault_injection"] = 0
         dest_params["manual_wal_flush_one_in"] = 0
+        dest_params["get_update_since_one_in"] = 0
         # Wide-column pessimistic transaction APIs are initially supported for
         # WriteCommitted only
         dest_params["use_put_entity_one_in"] = 0
+    if dest_params.get("get_update_since_one_in") != 0:
+        # To avoid sequence number hole in WAL
```

Comment:
I think log file recycling is also not compatible with `GetUpdatesSince` so maybe we can disable it here too. We should probably mention this in the documentation for `GetUpdatesSince`.